### PR TITLE
Visualization Update

### PR DESCRIPTION
**Pull request by RoboticWater**
Swapped out all the visualizations with ones that work with incidents. Now queries all reports/incidents in the last 48hrs. List of visualizations:
1. Report graph (read/unread reports)
2. Source breakdown (ct list)
3. Identified hate speech graph
4. Algorithm identified hate speech

### CODE REVIEW

File: lib/api/v1/visualization-controller.js [hunk 1]
```diff
@@ -3,91 +3,47 @@
...
 var _ = require("underscore");
 var writelog = require("../../writeLog");
 var tags = require("../../../shared/tags");
 var ReportQuery = require("../../../models/query/report-query");
 
 module.exports = function (app, user) {
-  app = app || express();
-  user = user || require("../authorization")(app);
+	app = app || express();
+	user = user || require("../authorization")(app);
```
Comment (cooperka) [inline lib/api/v1/visualization-controller.js]: Please change back to spaces not tabs :)

**Review by cooperka — APPROVED** [review]
Just a nit about spacing, otherwise the code LGTM. I just did a cursory glance, would love someone to test UI before merging.

File: public/angular/js/controllers/analysis/index.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  .controller("AnalysisController", [
+    "$scope",
+    "Socket",
+    "data",
+    "smtcTags",
+    function ($scope, Socket, data, smtcTags) {
+      var endDate = new Date();
+      var startDate = new Date();
+      startDate.setTime(startDate.getTime() - 2 * (24 * 60 * 60 * 1000));
+      var hateSpeechCutoff = 0.7;
```
Comment (harshilshah4251) [inline public/angular/js/controllers/analysis/index.js]: replace this with value from config.

Comment (RoboticWater) [reply public/angular/js/controllers/analysis/index.js]: The newest commit should fix this.

**Pull request merged**